src/d_iwad.c

```diff
@@ -196,105 +211,73 @@ static registry_value_t uninstall_values[] =
 
 // Values installed by the GOG.com and Collector's Edition versions
 
-static registry_value_t root_path_keys[] =
+static root_path_t gog_ce_paths[] =
```

Comment:
`static const` probably? And aren't these rather `gog_paths[]`? The Collector's Edition (i.e. `_ce`)  seems to be just one of them.

Reply:
Updated [here](https://github.com/fabiangreffrath/woof/pull/2260/commits/870e3b756e2323b98677447c92f50b4032a38604) and [here](https://github.com/fabiangreffrath/woof/pull/2260/commits/dbc191de20e1123d632ed7438d8454845ce2705a).